packages/edit-post/src/components/visual-editor/index.js

```diff
@@ -271,6 +285,7 @@ export default function VisualEditor( { styles } ) {
 			? { ...globalLayoutSettings, ...layout, type: 'constrained' }
 			: { ...globalLayoutSettings, ...layout, type: 'default' };
 	}, [
+		layout,
```

Comment:
Nit: eslint is reporting that if `layout` is introduced here in the dependency array, then we don't need `layout?.type`, `layout?.inherit`, `layout?.contentSize`, and `layout?.wideSize` in the array:

<img width="598" alt="image" src="https://user-images.githubusercontent.com/14988353/223579067-b6fea33f-4642-484a-89b3-2f85e7a7a2a3.png">

Reply:
Ha, I added "layout" to the dependency array because eslint was complaining that it was missing that dependency, and now it's complaining about not needing the others 🤦 
I think overall it might be better to just have the properties instead of the actual object, that way it might not re-run if it's a new object with unchanged properties? I'm not 100% sure how `useMemo` works.

Reply:
> I think overall it might be better to just have the properties instead of the actual object, that way it might not re-run if it's a new object with unchanged properties?

Same. We could keep it as it is on `trunk` and then look to open a tiny PR to explore the dependency array separately, if that simplifies things?

Reply:
Done!